extension/parquet/include/parquet_reader.hpp

```diff
@@ -63,9 +63,9 @@ class ParquetReader {
 	}
 
 	ParquetReader(ClientContext &context, string file_name, const vector<LogicalType> &expected_types_p,
-	              const string &initial_filename = string());
-	ParquetReader(ClientContext &context, string file_name)
-	    : ParquetReader(context, move(file_name), vector<LogicalType>()) {
+	              const string &initial_filename = string(), bool binary_as_string = false);
+	ParquetReader(ClientContext &context, string file_name, bool binary_as_string = false)
```

Comment:
Perhaps we could package this in a struct (`struct ParquetOptions`) so we can easily add more flags in the future?